microsoft/PowerToys
# [Shortcut Guide] Win key crash

## Summary of the Pull Request

**What is this about:**

**What is include in the PR:** 
- Return immediately if `WM_KEYDOWN` is not handled by our WndProc
- Remove `keyboard_state.h`

**How does someone test / validate:** 
- Start SG
- Press the Win key
- Verify that SG doesn't crash

## Quality Checklist


## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
